testui/support/testui_driver.py

```diff
@@ -6,11 +6,11 @@
 from pathlib import Path
 
 from appium.webdriver.common.touch_action import TouchAction
-from appium.webdriver.webdriver import WebDriver
 from selenium.webdriver import TouchActions, ActionChains
 from selenium.common.exceptions import WebDriverException
+from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
```

Comment:
Why do you create an alias instead of importing it as `WebDriver`?

Reply:
wanted to add some clarity, but rn it is impossible to do that. so this change is unnecessary.